src/clients/api/queries/getIsolatedPools/formatOutput/index.ts

```diff
@@ -71,6 +71,11 @@ const formatToPools = ({
         return acc;
       }
 
+      // Skip BIFI
+      if (vToken.address.toLowerCase() === '0xc718c51958d3fd44f5f9580c9ffac2f89815c909') {
```

Comment:
I'd recommend making this more generic and adding a comment to explain why we do that:

```typescript
import { MAINNET_TOKENS } from 'constants/tokens';
import { areTokensEqual } from 'utilities';

// Temporary hotfix following multichain issue
if (areTokensEqual(vToken.underlyingToken, MAINNET_TOKENS.bifi)) {
```

Reply:
Good call, made the change